ForkMe-Mobile/app/src/main/res/drawable/ic_fork.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="1024dp" android:viewportHeight="1024.0"
```

Comment:
I recommend changing the dimensions and viewport dimensions to match a typical icon. Won't make a difference if you explicitly specify the dimensions where ever you use this drawable, but just for completeness, I suggest that it be changed.